File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.js
function solution(a, b, c) {
    let mySet = new Set([a,b,c]);
    if (mySet.size === 1){
        return (a+b+c)*(a**2+b**2+c**2)*(a**3+b**3+c**3);
    }else if (mySet.size === 2){
        return (a+b+c)*(a**2+b**2+c**2);
    }else{
        return (a+b+c);
    }
}

File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.py
def solution(a, b, c):
    s3 = a + b + c
    s2 = s3 * (a**2 + b**2 + c**2)
    s1 = s2 * (a**3 + b**3 + c**3)
    unique = len(set([a, b, c]))
    if unique == 1: return s1
    if unique == 2: return s2 
    if unique == 3: return s3